crates/optimism/payload/src/payload.rs

```diff
@@ -33,12 +34,23 @@ pub struct OptimismPayloadBuilderAttributes {
     pub payload_attributes: EthPayloadBuilderAttributes,
     /// `NoTxPool` option for the generated payload
     pub no_tx_pool: bool,
-    /// Transactions for the generated payload
+    /// Decoded Transactions for the generated payload
     pub transactions: Vec<TransactionSigned>,
+    /// Encoded Transactions for the generated payload
+    pub encoded_transactions: Vec<Bytes>,
     /// The gas limit for the generated payload
     pub gas_limit: Option<u64>,
 }
 
+/// Optimism Payload Builder Attributes Helper
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct OptimismPayloadBuilderAttributesHelper {
+    /// An decoded signed transaction
+    pub decoded_transaction: TransactionSigned,
+    /// An encoded signed transaction
+    pub encoded_transaction: Bytes, 
+}
```

Comment:
cool, this is what we want,

I'd like to make this type reusable, like:

https://github.com/paradigmxyz/reth/blob/5e1b0fb96d1b6dbc56532deaeab4dca934729f3e/crates/net/peers/src/lib.rs#L200-L202

`WithEncoded<>(Bytes, T)`

we can add this helper type to reth-primitives/transactions

Reply:
Currently working on this.

I'm going to put it in `reth-primitives/transactions/mod.rs` unless you think `reth-primitives/transactions/optimism.rs` is a better place for it?

Reply:
> reth-primitives/transactions/mod.rs

this seems fine

Reply:
ok thank you for the guidance

Reply:
I made an attempt at this